install.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+mv example.env .env
+
+echo "Creating venv..."
+python3 -m venv venv
+
+echo "Activating venv..."
+source ./venv/Scripts/activate
```

Comment:
One thing here is that if, for some reason (should be rare), this fails (say `python3 -m venv` is not available on the machine), then we go ahead and install the requirements anyway, we could maybe error or prompt the user in that case.

Reply:
If we do not want to require users to use venv, I think prompt for global installation is ok